types/mongodb/index.d.ts

```diff
@@ -36,6 +37,8 @@ import { EventEmitter } from 'events';
 import { Readable, Writable } from "stream";
 import { checkServerIdentity } from "tls";
 
+type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
We can maybe add a comment to indicate that this is needed until the typescript version is lower than 3.5

Reply:
Done